examples/data-sources/b1ddi_dns_views/data-source.tf

```diff
@@ -1,18 +1,24 @@
 terraform {
   required_providers {
     b1ddi = {
-      version = "0.1"
       source  = "infobloxopen/b1ddi"
     }
   }
 }
 
 # Select DNS View by name
-data "b1ddi_dns_views" "example_dns_view" {
+data "b1ddi_dns_views" "tf_example_dns_views" {
   filters = {
-    "name" = "example_tf_dns_view"
+    "name" = "tf_example_dns_views"
```

Comment:
Same as above.

Reply:
Reverted it back to "example_tf_dns_view"